README.md

```diff
@@ -521,7 +524,61 @@ mailersend.domainSettings({
     console.log(data);
   });
 ```
+### Add a domain 
 
+```js
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.addDomain({
+    name: "example.com",
+    return_path_subdomain: "rp_subdomain",
+    custom_tracking_subdomain: "ct_subdomain",
+    inbound_routing_subdomain: "ir_subdomain",
+  })
+  .then((response) => response.json())
+  .then((data) => {
+    console.log(data);
+  });
+```
+
+### Get DNS records 
+
+```js
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.getDNS({
+    domain_id: "xxx",
+  })
+  .then((response) => response.json())
+  .then((data) => {
+    console.log(data);
+  });
+```
```

Comment:
Please add an empty new line after this block.

Reply:
done